client/src/components/map.tsx

```diff
@@ -7,89 +7,99 @@ import VectorLayer from 'ol/layer/Vector';
 import VectorSource from 'ol/source/Vector';
 import {fromLonLat} from 'ol/proj';
 import * as styles from './map.module.scss';
-import olms from 'ol-mapbox-style';
+// import olms from 'ol-mapbox-style';
```

Comment:
Is there a reason this is commented out?

Reply:
This was formerly WIP; I can remove comments for clarity going forward as we get this merge-ready